addons/website/views/snippets/snippets.xml

```diff
@@ -1277,6 +1277,8 @@
                             data-use-css-color="true" data-selected-tab="custom-colors"/>
             <we-select data-img="/website/static/src/img/snippets_options/palette.svg" class="o_we_theme_colors_select" data-variable="color-palettes-name"/>
         </we-row>
+        <we-collapse class="o_we_theme_presets_collapse" string="Color Presets">
+        </we-collapse>
```

Comment:
Note: in XML you could auto close that tag (not required, just for info in case you did not know)